lib/cursor/Taskfile.yml

```diff
@@ -0,0 +1,73 @@
+version: "3"
+
+tasks:
+  fetch-rule:
+    desc: |
+      Fetch a single file from a private GitHub repo.
+      Example: `task cursor:fetch-rule -- cursor_rules.mdc`
+    vars:
+      OWNER: masterpointio
+      REPO: internal-cursor-rules
+      REF: main
+      OUTDIR: .cursor/rules
```

Comment:
Let's move these vars to a global level, as they are used in `fetch-rules-all` as well.

Reply:
Good call. Moved these to global vars within the file.

Reply:
Mind address the lint issue? I'll approve ASAP then.

Reply:
Ah, sorry. I didn't see the linting errors during the git commit + push locally. Resolved them and good to go.